tests/test_functional.py

```diff
@@ -0,0 +1,56 @@
+import torch
+from fastai.data.all import test_eq as _test_eq
+from unittest.mock import patch
+
+from fastaudio.functional import region_mask
+
+
+class TestCreateRegionMask:
```

Comment:
This unittest style class can probably be transformed to the pytest style of simple functions

Reply:
Are we preferring simple function tests of the class style?

Reply:
Yes